src/Features/Core/Portable/Diagnostics/AnalyzerHelper.cs

```diff
@@ -356,6 +356,9 @@ private static IEnumerable<AnalyzerPerformanceInfo> Convert(IEnumerable<(Diagnos
             Contract.ThrowIfFalse(project.SupportsCompilation);
             AssertCompilation(project, compilation);
 
+            // Always run diagnostic suppressors.
+            analyzers = AppendDiagnosticSuppressors(analyzers, allAnalyzersAndSuppressors: service.GetDiagnosticAnalyzers(project));
```

Comment:
it should give in filteredAnalyzers.

Reply:
I think given analyzers will included all suppressors?

Reply:
Ah, I was not sure so just being more conservative here. I can convert this into an assert if you confirm that is the case for all current callers.